### PR TITLE
Remarks Glossary

### PR DESCRIPTION
**Pull request by mjoyce91**
Static UI only

Screen Shot 2022-03-23 at 10 40 02 AM

### CODE REVIEW

File: src/Components/Agenda/AgendaItemResearchPane/RemarksGlossary/RemarksGlossary.jsx [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+                      <FA {...faProps} />
+                    </InteractiveElement>
+                    <span className="remark-text">{r.rmrktext}</span>
+                    {
+                      hasTextInput &&
+                      <TextInput
+                        value={getTextInputValue(r.rmrkseqnum)}
+                        changeText={v => setTextInput(r.rmrkseqnum, v)}
+                        customContainerClass="remarks-input-container"
+                        inputProps={{ autocomplete: 'off' }}
```
Comment (mjoyce91) [inline src/Components/Agenda/AgendaItemResearchPane/RemarksGlossary/RemarksGlossary.jsx:72]: need to change to `autoComplete`

File: src/sass/_agendaItemMaintenance.scss [hunk 2]
```diff
@@ -296,3 +296,98 @@
...
+  .remark-category {
+    background-color: $color-cornflower-blue;
+    color: $color-white;
+    border-radius: 5px;
+    padding: .5em;
+    font-size: 13px;
+    display: flex;
+  }
+
+  @each $remark-id, $color in $remarks {
```
Comment (elizabeth-jimenez) [inline src/sass/_agendaItemMaintenance.scss:387]: ❤️

File: src/Components/Agenda/AgendaItemResearchPane/RemarksGlossary/RemarksGlossary.jsx (hunk 1 above)
Comment (elizabeth-jimenez) [reply src/Components/Agenda/AgendaItemResearchPane/RemarksGlossary/RemarksGlossary.jsx:72]: ouch, too much typing - 🎹

**Review by elizabeth-jimenez — APPROVED** [review]
gucci

File: src/sass/_agendaItemMaintenance.scss (hunk 2 above)
Comment (elizabeth-jimenez) [reply src/sass/_agendaItemMaintenance.scss:387]: def stealing this 🏦

**Pull request merged**